web/index.html

```diff
@@ -3,6 +3,17 @@
 <head>
     <meta charset="utf-8" />
     <title>Expensify.cash</title>
+    <meta name="description" content="Chat and payments are now the same thing. Launching Summer 2021, join the waitlist to be first in line!">
+    <!-- Twitter -->
+    <meta property="twitter:card" content="summary">
+    <meta property="twitter:site" content="@expensify">
+    <!-- Open Graph / Facebook -->
```

Comment:
Same, I think we can get rid of this comment.

Reply:
sure